From 165e7ad459fbba2f89708fba04a55bb3981e884c Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <szabolcs.nagy@arm.com>
Date: Wed, 2 Feb 2022 14:03:58 +0000
Subject: [PATCH 158/423] Fix elf/tst-audit25a with default bind now toolchains

This test relies on lazy binding for the executable so request that
explicitly in case the toolchain defaults to bind now.

(cherry picked from commit 80a08d0faa9b224019f895800c4d97de4e23e1aa)
---
 elf/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elf/Makefile b/elf/Makefile
index 8a524a7f6a..89021fb4dc 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -2089,6 +2089,7 @@ $(objpfx)tst-audit25a: $(objpfx)tst-audit25mod1.so \
 		       $(objpfx)tst-audit25mod2.so \
 		       $(objpfx)tst-audit25mod3.so \
 		       $(objpfx)tst-audit25mod4.so
+LDFLAGS-tst-audit25a = -Wl,-z,lazy
 $(objpfx)tst-audit25mod1.so: $(objpfx)tst-audit25mod3.so
 LDFLAGS-tst-audit25mod1.so = -Wl,-z,now
 $(objpfx)tst-audit25mod2.so: $(objpfx)tst-audit25mod4.so
-- 
2.30.2

